provisioning/roles/mailcatcher/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+- name: ensure needed packages are installed
+  apt: pkg={{ item }} state=latest
+  sudo: yes
+  with_items:
+    - libsqlite3-dev
+
+- name: ensure mailcatcher installed
+  gem: name=mailcatcher user_install=no state=latest
+  sudo: yes
```

Comment:
I think you have to tell supervisord to run the binary manually the first time.

I know it isn't done in the `solr` role, but AFAIK it was never tested.

We will have to use solr next week for a project, if you want I can take care of that myself once I know how to do it.